.env.example

```
@@ -26,3 +26,55 @@ VOYAGE_API_KEY=
+# ====================================================================
+# DOCKER CONFIGURATION
+# ====================================================================
+
+# Build target: "development" or "production"
+BUILD_TARGET=development
+
+# Backend port
+BACKEND_PORT=8000
+
+# Frontend ports
+FRONTEND_DEV_PORT=5173  # Vite dev server (development)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)